lib/py/src/transport/TSocket.py

```diff
@@ -159,6 +159,10 @@ def __init__(self, host=None, port=9090, unix_socket=None, socket_family=socket.
         self._unix_socket = unix_socket
         self._socket_family = socket_family
         self.handle = None
+        self._backlog = 128
+
+    def setBacklog(self, backlog):
```

Comment:
What happens when you call this when it is already listening?

Reply:
Yeah, nothing works except `_backlog` changed. I should add some warning for it. Thank for your suggestions!